### ISSUE TITLE
Batch norm layer fails when batch_size is set to 1

### ISSUE THREAD

**Issue by johntiger1** [cs230-stanford/cs230-code-examples#19]
When running the code, I found that this particular layer (and only this one) fails when the batch size is set to 1. 

https://github.com/cs230-stanford/cs230-code-examples/blob/159df10a6187c7e1d6ec949c8e06d7f67f8f1cd2/pytorch/vision/model/net.py#L78
